src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ProcessorService.cs

```diff
@@ -119,5 +120,13 @@ public async Task ProcessStage4Messages(ImportMessage message, ICollector<Import
 
             await statisticsDbContext.SaveChangesAsync();
         }
+
+        private async Task<SubjectData> GetSubjectDataFromMainDataFile(ImportMessage message)
+        {
+            var dataFileBlobPath = AdminReleasePath(message.Release.Id, ReleaseFileTypes.Data, message.DataFileName);
+
+            var subjectData = await _fileStorageService.GetSubjectData(message.Release.Id, dataFileBlobPath);
```

Comment:
May as well just return here but also see my other comment about removing FileStorageService in favour of BlobStorageService.

Reply:
Yup agreed wrt phasing out FileStorageService.  Do we have a ticket for this?

Reply:
Created https://dfedigital.atlassian.net/browse/EES-1743 thanks.